### PR TITLE
Finished implementation

### PR DESCRIPTION
**Pull request by akselleirv**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 description: |-
-  
+  The Azure PIM provider was built as PIM group eligible assignment is as of writing not supported in the official azurerm provider.
 ---
 
-# scaffolding-framework Provider
-
+# azurepim Provider
 
+The Azure PIM provider was built as PIM group eligible assignment is as of writing not supported in the official azurerm provider.
```
Comment (nikolaifa) [inline docs/index.md:11]:
```suggestion
The Azure PIM provider was built as PIM group eligible assignment is as of writing not supported in the official azuread provider.
```

File: docs/index.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ---
 # generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "scaffolding-framework Provider"
+page_title: "azurepim Provider"
 subcategory: ""
 description: |-
-  
+  The Azure PIM provider was built as PIM group eligible assignment is as of writing not supported in the official azurerm provider.
```
Comment (nikolaifa) [inline docs/index.md:6]:
```suggestion
  The Azure PIM provider was built as PIM group eligible assignment is as of writing not supported in the official azuread provider.
```

File: examples/resources/group_eligible_assignment/dev.tfrc [hunk 3]
```diff
@@ -1,7 +1,7 @@
 provider_installation {
 
   dev_overrides {
-      "akselleirv/azurepim" = "/Users/t971059/go/bin"
+      "TelenorNorway/azurepim" = "/Users/t971059/go/bin"
```
Comment (nikolaifa) [inline examples/resources/group_eligible_assignment/dev.tfrc:4]: Lets avoid our user identities
```suggestion
      "TelenorNorway/azurepim" = "~/go/bin"
```

File: docs/resources/group_eligible_assignment.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "azurepim_group_eligible_assignment Resource - terraform-provider-azurepim"
+subcategory: ""
+description: |-
+  Manages an Entra Group PIM Eligible Role Assignment.
+---
+
+# azurepim_group_eligible_assignment (Resource)
+
+Manages an Entra Group PIM Eligible Role Assignment.
```
Comment (nikolaifa) [inline docs/resources/group_eligible_assignment.md:11]:
```suggestion
Enables PIM for an Entra group, manages an PIM Eligible Role Assignment and sets the PIM policy for the member role to allow for no expiration on eligible assignments.  
```

I'd like to mention this one

Comment (akselleirv) [reply docs/resources/group_eligible_assignment.md:11]: Added this to the go code where this markdown is generated from.

**Review by nikolaifa — APPROVED** [review]
LGTM

**Pull request merged**